### PR TITLE
feat(ClipboardCopy): remove switchDelay prop

### LINKED ISSUE #224: Clipboard copy - switchDelay prop
Follow up to breaking change issue https://github.com/patternfly/patternfly-react/issues/8589
PR: https://github.com/patternfly/patternfly-react/pull/8619
Switch delay prop should be removed.

_Required actions:_
1. Build codemod
2. Build test
3. Update readme with description & example

### PR DESCRIPTION
**Pull request by gitdallas**
closes #224

### CODE REVIEW

File: packages/pf-codemods/README.md [hunk 1]
```diff
@@ -174,6 +174,24 @@ Out:
 <ClipboardCopyButton position="bottom" />
 ```
 
+### clipboardCopy-remove-popoverPosition [(#8619)](https://github.com/patternfly/patternfly-react/pull/8619)
```
Comment (thatblindgeye) [inline packages/pf-codemods/README.md:177]:
```suggestion
### clipboardCopy-remove-switchDelay [(#8619)](https://github.com/patternfly/patternfly-react/pull/8619)
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

**Review by thatblindgeye — APPROVED** [review]

File: packages/pf-codemods/README.md [hunk 2]
```diff
@@ -192,6 +192,24 @@ Out:
 <CodeEditor      />
 ```
 
+### clipboardCopy-remove-switchDelay [(#8619)](https://github.com/patternfly/patternfly-react/pull/8619)
+
+We've removed the switchDelay prop the `ClipBoardCopy` component.
```
Comment (jenny-s51) [inline packages/pf-codemods/README.md:197]:
```suggestion
We've removed the switchDelay prop from the `ClipBoardCopy` component.
```

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
LGTM @gitdallas , just needs one tiny update to the README

File: packages/pf-codemods/README.md (hunk 2 above)
Comment (gitdallas) [reply packages/pf-codemods/README.md:197]: man idk how i ended up with so many typos on this one

**Review by jenny-s51 — APPROVED** [review]

**Pull request merged**